source/app.d

```diff
@@ -114,7 +114,7 @@ void main()
 	bool noMonitoring;
 	setLogFile("log.txt", LogLevel.diagnostic);
 
-    version (linux)
+    version (linux) version (DMD)
```

Comment:
Personally I went with a slightly different approach: https://github.com/dlang/dlang-bot/commit/9b0e9cb6b142b8c660c659bfa32bc31e5ae23365

Reply:
For me this resulted in a linker error rather than a compile error, so the `typeof` wouldn't have helped. Maybe this has changed in a recent LDC version?